browser/src/control/Control.PartsPreview.js

```diff
@@ -499,6 +499,11 @@ L.Control.PartsPreview = L.Control.extend({
 	},
 
 	_setPart: function (e) {
+		if (cool.Comment.isAnyEdit()) {
```

Comment:
All of these different actions from different components are using `this._map.setPart` in the end. Could it be possible to fix it in one place there?

Or these actions have other side effects that could make it more complicated?

Reply:
It makes it more complicated as these methods are calling other functions like selectPart and deselectAll or _scrollViewToPartPosition. I am afraid I might go into a rabbit hole if we don't stop these actions at their starting point.

Reply:
I see